platform/security/core/security-core-impl/src/main/java/ddf/security/service/impl/SecurityAssertionStore.java

```diff
@@ -48,26 +47,24 @@
     public static SecurityAssertion getSecurityAssertion(Message message) {
         if (message != null) {
             TokenStore tokenStore = getTokenStore(message);
-            Principal principal = (Principal) message.get(WSS4JInInterceptor.PRINCIPAL_RESULT);
```

Comment:
Actually found another way to grab the Principal from the message and only use the WsEngineResults as a backup, so functionality restored :)